integration-cli/docker_cli_proxy_test.go

```diff
@@ -40,24 +38,15 @@ func (s *DockerDaemonSuite) TestCliProxyProxyTCPSock(c *check.C) {
 		}
 	}
 
-	if ip == "" {
-		c.Fatal("could not find ip to connect to")
-	}
-
-	if err := s.d.Start("-H", "tcp://"+ip+":2375"); err != nil {
-		c.Fatal(err)
-	}
-
+	c.Assert(ip, checker.Equals, "")
```

Comment:
Should be `c.Assert(ip, checker.Not(checker.Equals), "")`, because we want to make sure it is not empty (and not the other way around :wink:.